include/pinocchio/algorithm/default-check.hpp

```diff
@@ -12,9 +12,10 @@
 namespace pinocchio
 {
   /// Default checker-list, used as the default argument in Model::check().
-  inline AlgorithmCheckerList<ParentChecker, CRBAChecker, ABAChecker> makeDefaultCheckerList()
+  inline AlgorithmCheckerList<ParentChecker, CRBAChecker, ABAChecker, MimicChecker>
+  makeDefaultCheckerList()
   {
-    return makeAlgoCheckerList(ParentChecker(), CRBAChecker(), ABAChecker());
+    return makeAlgoCheckerList(ParentChecker(), CRBAChecker(), ABAChecker(), MimicChecker());
```

Comment:
I don't think this should be added to the default checker list.
Default checker validate if the model is correct, Mimic doesn't imply the model is not valid, it only doesn't work on some algorithms…